src/Microsoft.AspNet.Mvc.Core/TaskCache.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNet.Mvc
+{
+    public static class TaskCache
+    {
+#if DNX451 || DNX452
+        /// <summary>A task that's already been completed successfully.</summary>
+        private static Task s_completedTask;
```

Comment:
You can put it as it is now or change some names. 

The reason i suggested was just because it looks more readable!